variables.tf

```diff
@@ -1,13 +1,28 @@
 variable "auth_token" {
+  description = "This is your Packet API Auth token. This can also be specified with the TF_VAR_PACKET_AUTH_TOKEN shell environment variable."
```

Comment:
Equinix Metal / TF_VAR_METAL_AUTH_TOKEN

Reply:
Fixed this. Got rid of the export verbiage as well. Not really needed.